### PR TITLE
network update: sprint 46

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: blog/2023-10-02-network.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+We started reviewing the bootstrap peers PR, [ouroboros-network#4615].
+
+### Towards Typed Protocols 0.2.0.0
+
+We discovered a performance regression when using `typed-protocols-0.2.0.0`, and we
+started investigating where it comes from.  Currently, we see that
+`typed-protocols-0.2.0.0` can outperform `typed-protocols-0.1.0.0` when running
+in isolation with a simple `ping-pong` protocol, so the regression might be in
+the new block fetch implementation which comes with `typed-protocols-0.2.0.0`
+See
```
Comment (karknu) [inline blog/2023-10-02-network.md:24]: See?
Is there a missing reference?

File: blog/2023-10-02-network.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+We merged two PRs written by Galois engineers:
+
+* a pull request which refactors the main entry function for P2P, see [ouroboros-network#3834]; 
+* a pull request which reviews usage of `unsafe` function in the network code based.
+
+### IO-Sim
+
+#### IOSimPOR
+We found and fixed a bug in `IOSimPOR`.  We'd like to thank Prof. John Hughes
+(Quviq AB) for helping us with debugging the issue.
```
Comment (karknu) [inline blog/2023-10-02-network.md:37]: I think Quviq is misspelled.

File: blog/2023-10-02-network.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+To prepare for the next release, we consolidate packages taking advantage of
+the public sublibraries supported now both by `cabal` and `Hackage`.   This is
+a work in progress, [io-sim#114].
+
+### P2P adoption
+
+In the last two weeks, we've seen increase in P2P adoption.  Karl Knutsson
+([Cardano Foundation][CF]) is investigating this issue.
+![P2P relays](/images/network/2023-10-04-p2p-relays.png)
```
Comment (karknu) [inline blog/2023-10-02-network.md:63]: I'm not really looking at the increase in P2P adoption.

**Review by karknu — COMMENTED** [review]
Just some minor things.

File: blog/2023-10-02-network.md (hunk 3 above)
Comment (coot) [reply blog/2023-10-02-network.md:63]: Right, leftover from last sprint.

File: blog/2023-10-02-network.md (hunk 2 above)
Comment (coot) [reply blog/2023-10-02-network.md:37]: I think it's right, I don't see difference from how you spelled it.

File: blog/2023-10-02-network.md (hunk 1 above)
Comment (coot) [reply blog/2023-10-02-network.md:24]: yes

**Comment by coot** [comment]
@karknu addressed in #265.

**Pull request merged**